packages/core/src/common/_customs.scss

```diff
@@ -158,7 +158,15 @@ $pt-dark-elevation-shadow-0: 0 0 0 1px transparent,
                              0 0 0 rgba($black, 0),
                              0 0 0 rgba($black, 0) !default;
 
-// Custom colors with shades:
+// Custom colors without shades:
+
+$g-dark-orange: #FF7F0E;
+$g-orange: tint($g-dark-orange, 20%);
+$g-dark-wizard-10: #2E6537;
+$g-wizard-10: tint($g-dark-wizard-10, 20%);
```

Comment:
I spoke with Guillermo and he changed them to be more consistent with what we have.

So the colors are now the following: 
- warning: orange600
- success: grass60
- danger: error600
- primary: primary600

I think the primary is not added. I also left a [comment in this PR](https://github.com/graphext/blueprint/pull/61#discussion_r924262871). If you can add the primary colors, it would be perfect 😍